pkg/network/node/node.go

```diff
@@ -332,7 +335,9 @@ func (node *OsdnNode) Start() error {
 	if err != nil {
 		return err
 	}
-
+	if err = node.egressIP.Start(node.networkClient, nodeIPTables); err != nil {
```

Comment:
We should not call this for single-tenant/subnet plugin as it depends on NetNamespaces.
I think, we should move this to node.policy.StartEgressIP?